dqc/system/sol.py

```diff
@@ -240,6 +242,23 @@ def requires_grid(self) -> bool:
     def getparamnames(self, methodname: str, prefix: str = "") -> List[str]:
         pass
 
+    def make_copy(self, **kwargs) -> "Sol":
```

Comment:
Thank you for doing this with `Sol` as well. This is still an experimental feature that is not as developed as `Mol`.
I have the same comment with `Mol` above.

Reply:
added